README.md

```diff
@@ -66,11 +66,106 @@ Select desired `Start` and `End` waypoints using the `RMF Panel` and click the `
 ```bash
 source ~/rmf_demos_ws/install/setup.bash
 ros2 launch demos office_loop.launch.xml
-``` 
+```
 
 ![](docs/media/loop_request.gif)
 
----
+#### Secure Ros2 Office Demo
+
+The office demo can be run in secure mode using the [ROS 2 DDS-Security integration](https://design.ros2.org/articles/ros2_dds_security.html) (SROS2) capabilities. This security features will provide encryption, authentication and access control to the whole ROS2 setup of RMF.
+
+To use ROS2 to secure the office demo a few environmental variables need to be set. Adding them to a script would make it easier to source them in any shell where we need to run ros2 secured nodes.
+
+```bash
+echo 'export ROS_SECURITY_KEYSTORE=~/rmf_demos_ws/keystore
+export ROS_SECURITY_ENABLE=true
+export ROS_SECURITY_STRATEGY=Enforce
+export ROS_DOMAIN_ID=42' > sros2_environment.sh
```

Comment:
Should we add `export RMW_IMPLEMENTATION=rmw_cyclonedds_cpp` here and bring the Cyclone DDS installation instruction to above this instruction block?

Reply:
Also explicitly have instructions to clone the cyclone DDS repos into the workspace and checkout the `foxy` branch

Reply:
Okay I understand now that we cannot have the`RMF_IMPLEMENTATION` line inside this file at this point because we are relying on FastDDS to generate the  security artifacts.